src/encoded/static/components/award.js

```diff
@@ -409,24 +737,481 @@ class StatusChart extends React.Component {
     }
 }
 
-StatusChart.propTypes = {
+BiosampleChart.propTypes = {
     award: PropTypes.object.isRequired, // Award being displayed
-    statuses: PropTypes.array, // Array of status facet data
-    linkUri: PropTypes.string.isRequired, // URI to use for matrix links
+    categoryData: PropTypes.array.isRequired, // Type-specific data to display in a chart
+    categoryFacet: PropTypes.string.isRequired, // Add to linkUri to link to matrix facet item
     ident: PropTypes.string.isRequired, // Unique identifier to `id` the charts
+    linkUri: PropTypes.string.isRequired,
+    selectedOrganisms: PropTypes.array,
 };
 
-StatusChart.defaultProps = {
-    statuses: [],
+BiosampleChart.defaultProps = {
+    selectedOrganisms: [],
 };
 
-StatusChart.contextTypes = {
+BiosampleChart.contextTypes = {
+    navigate: PropTypes.func,
+};
+
+// Function to be called in createChart and updateChart of class StatusExperimentChart, creates arrays for chart data
+function StatusData(experiments, unreplicated, isogenic, anisogenic) {
+    let unreplicatedArray;
+    let isogenicArray;
+    let anisogenicArray;
+    const unreplicatedLabel = [];
+    let unreplicatedDataset = [];
+    const isogenicLabel = [];
+    let isogenicDataset = [];
+    const anisogenicLabel = [];
+    let anisogenicDataset = [];
+
+    if (experiments && experiments.facets && experiments.facets.length) {
```

Comment:
this whole StatusData method is pretty tortured but I blame the data model, not you.  maybe we need a bigger datastructure  instead of 3 differernt named ones?